docs/src/views/ThemingExamples.tsx

```diff
@@ -330,7 +359,23 @@ export default () => (
 
           <Provider
             theme={{
-              siteVariables: { brand: 'darkred' },
+              componentVariables: {
+                Button: {
+                  primaryBackgroundColor: 'darkred',
+                },
+              },
+              componentStyles: {
+                Button: {
+                  root: ({
+                    variables: v,
+                    props: p,
+                  }: ComponentStyleFunctionParam<any, ThemingExamplesVariables>) => ({
+                    ...(p.primary && {
```

Comment:
You do not need the componentStyles here again, as those are not overriding anything from before. Just the componentVariables part should be enough